### ISSUE TITLE
Category live chat

### ISSUE THREAD

**Issue by CronaTronic** [Lior-za/Soen341UC3#12]
As a user, I want to be able to communicate with others in real time about the topics I'm asking questions and providing answers to.

**Comment by I-Dream-Of-Machines** [comment]
Do we still want to have this feature? I am unsure about whether this really adds to our website given the amount of work that might be required to get this going. I think features such as the search and the categorizing might be off more value @CronaTronic @Lior-za

**Comment by CronaTronic** [comment]
I don't think so. We can throw it back onto the backlog and pull it off if we find the time
